deployment.yml

```diff
@@ -0,0 +1,39 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+    name: wallet-api
+spec:
+    selector:
+        matchLabels:
+            app: wallet-api
+    replicas: 2
```

Comment:
your deployment currently runs 2 replicas. this wasn't required. please change to 1. this will also make it easier for you to check for running pods.